eng/common/TestResources/New-TestResources.ps1

```diff
@@ -62,7 +62,10 @@ param (
     [switch] $CI = ($null -ne $env:SYSTEM_TEAMPROJECTID),
 
     [Parameter()]
-    [switch] $Force
+    [switch] $Force,
+
+    [Parameter()]
+    [switch] $OutFile
```

Comment:
I suspect we should probably make this default to true or default to $IsWindows as folks will most likely always want this especially if we end up integrating into our test frameworks which we I believe we should.

Reply:
Yes, when we get the test framework integration we should definitely default this to true.